src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -842,6 +843,10 @@ class ApplicationManagerImpl
     return resume_ctrl_;
   }
 
+  HmiInterfaces& hmi_interfaces() OVERRIDE {
```

Comment:
@dtrunov such interface brakes encapsulation. So please either return const reference or do no return this class at all